release/train_tests/benchmark/image_classification/factory.py

```diff
@@ -88,11 +91,26 @@ def collate_fn(self, batch):
         return batch["image"], batch["label"]
 
 
+class ImageClassificationTorchDataLoaderFactory(TorchDataLoaderFactory):
+    """Factory for creating PyTorch DataLoaders for image classification tasks."""
+
+    def __init__(self, benchmark_config: BenchmarkConfig):
+        train_urls = IMAGENET_PARQUET_SPLIT_S3_DIRS["train"]
+        val_urls = IMAGENET_PARQUET_SPLIT_S3_DIRS["train"]
```

Comment:
Should we just use `["val"]` here, instead of using `["train"]` with a limit?

Reply:
maybe it's a question for @justinvyu , since the existing ray data benchmark does the same

Reply:
Oh, the problem was that the `"val"` files do not include the label which is needed for validation. The labels need to be added as a column.

Reply:
I do a limit on the train set instead: https://github.com/ray-project/ray/blob/master/release/train_tests/benchmark/image_classification/factory.py#L56